daml-lf/encoder/src/test/lf/Interface_1.dev_.lf

```diff
@@ -15,8 +27,8 @@ module InterfaceMod {
       , controllers Cons @Party [call_method @InterfaceMod:Boxy getParty this] (Nil @Party)
       , observers Nil @Party
       to upure @Int64 i;
-    coimplements Other:Package {
-      method getParty = Other:Package {party} this;
+    coimplements OtherMod:Parcel {
```

Comment:
turns out this didn't typecheck!
also, I confused myself a bit with the name `Package`, so now it's `Parcel`

Reply:
:+1: